### PR TITLE
feat: UI of admin permissions page

### LINKED ISSUE #159: Add admin/permissions page (UI only)
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

image
located at `admin/permissions`

Please go through the details of #153 before starting to work on this.

### PR DESCRIPTION
**Pull request by harshitagupta30**
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

This PR adds the user interface for the admin's permission page.

Deployment link: https://open-event-frontend-harshita.herokuapp.com/admin/permissions

Fixes #159 

@niranjan94 @geekyd @CosmicCoder96 please review :+1:

### CODE REVIEW

File: app/routes/admin/permissions.js [hunk 1]
```diff
@@ -3,4 +3,7 @@ import Ember from 'ember';
 const { Route } = Ember;
 
 export default Route.extend({
+  titleToken() {
```
Comment (geekyd) [inline app/routes/admin/permissions.js]: Please remove this will be taken care of in #188.

File: app/templates/admin/permissions/event-roles.hbs [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    <tr>
+      <th>{{t 'Event Role/Service'}}</th>
+      <th>{{t 'Track'}}</th>
+      <th>{{t 'Session'}}</th>
+      <th>{{t 'Speaker'}}</th>
+      <th>{{t 'Sponsor'}}</th>
+      <th>{{t 'Microlocation'}}</th>
+    </tr>
+  </thead>
+  <tbody>
```
Comment (niranjan94) [inline app/templates/admin/permissions/event-roles.hbs]: Generate this whole table via a loop or something. All the data is repetitive

**Comment by harshitagupta30** [comment]
@niranjan94 @geekyd I have made suggested changes. 
Deployment link : https://open-event-frontend-harshita.herokuapp.com/admin
Please review it again. Thanks.

File: app/templates/admin/permissions.hbs [hunk 3]
```diff
@@ -1 +1,11 @@
+<div>
+  {{#tabbed-navigation}}
```
Comment (CosmicCoder96) [inline app/templates/admin/permissions.hbs]: @harshitagupta30 Please use the menu similar in design to the `admin/settings` route for this sub menu.
image
You can achieve it via the non pointing version of tabbed-navigation

**Comment by harshitagupta30** [comment]
@CosmicCoder96 updated!
@niranjan94 @geekyd Please review it again :+1:

File: app/templates/admin/permissions/event-roles.hbs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            <br>
+            {{ui-checkbox label=(t 'Delete') id=(concat 'delete_' number)}}
+          </td>
+        {{/each}}
+      </tr>
+    {{/each}}
+  </tbody>
+</table>
+<div>
+  <button class="ui teal button">{{t 'Save'}}</button>
```
Comment (harshitagupta30) [inline app/templates/admin/permissions/event-roles.hbs]: @CosmicCoder96 I've added the button you can see the code! It's on left most corner aligned with the form

Comment (CosmicCoder96) [inline app/templates/admin/permissions/event-roles.hbs]: @harshitagupta30 Oh, I checked the deployment link, wasn't in there at that time.Please update it 
image

**Comment by niranjan94** [comment]
@geekyd no. Let it not be like that. We may have more distinct pages here. Let it be `admin/permissions/system-roles` and `/admin/permissions/event-roles`

File: app/routes/admin/permissions.js [hunk 5]
```diff
@@ -5,5 +5,8 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('Permissions');
+  },
+  beforeModel() {
```
Comment (niranjan94) [inline app/routes/admin/permissions.js]: Create a `admin/permissions/index` route and put this `beforeModel` there instead. Then this will work fine

**Comment by harshitagupta30** [comment]
@niranjan94 @geekyd @CosmicCoder96 I've made all required changes. Please review it again.

File: app/routes/admin/permissions/index.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  beforeModel() {
+    return this.replaceWith('admin.permissions.system-roles');
```
Comment (geekyd) [inline app/routes/admin/permissions/index.js]: Please use `this.transitionTo` here. Check this https://guides.emberjs.com/v2.11.0/routing/redirection/

Comment (harshitagupta30) [inline app/routes/admin/permissions/index.js]: updated :+1:

**Pull request merged**